assets/js/src/core/components/upload/upload-list/upload-list.tsx

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
I think we can create a story for this? 
Or is the behavior of the upload still open for discussion?